lib/entities/team-membership.ts

```diff
@@ -69,25 +73,23 @@ export interface TeamMembership extends TeamMembershipProps, DefaultElements<Tea
 }
 
 function createTeamMembershipApi(http: AxiosInstance) {
+  const getParams = (data: TeamMembershipProps) => ({
+    teamMembershipId: data.sys.id,
+    teamId: data.sys.team.sys.id,
+    organizationId: data.sys.organization.sys.id,
+  })
+
   return {
     update: function () {
       const raw = this.toPlainObject()
-      const teamId = raw.sys.team.sys.id
-      return http
-        .put('teams/' + teamId + '/team_memberships/' + this.sys.id, raw, {
-          headers: {
-            'X-Contentful-Version': this.sys.version || 0,
-          },
-        })
-        .then((response) => wrapTeamMembership(http, response.data), errorHandler)
+      return endpoints.teamMembership
+        .update(http, getParams(raw), raw)
+        .then((data) => wrapTeamMembership(http, data))
     },
 
-    delete: function () {
+    delete: function del() {
       const raw = this.toPlainObject()
```

Comment:
i believe should be: `const raw = this.toPlainObject() as TeamMembershipProps`

Reply:
good call. fixed.